src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -70,7 +70,7 @@ class TextToMiniTest {
             |ERROR: Line 4: Column 8: Expected 't', but ' '
             |  impor t B []
             |       ^
-            |""".stripMargin.stripPrefix("\n").stripSuffix("\n"),
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
```

Comment:
Thanks for spotting this issue, but this change is still not correct. The newline characters put by the multiline string don't depend on the platform but on the text editor. I can still put `\r\n` even in mac, where you can still put `\n` in windows, depending on the setting of your text editor.

Just to make sure, did your tests fail without this change?